packages/@divvi/mobile/src/app/App.tsx

```diff
@@ -88,7 +89,11 @@ export class App extends React.Component<Props> {
 
   render() {
     return (
-      <SafeAreaProvider>
+      <SafeAreaProvider
+        // https://github.com/AppAndFlow/react-native-safe-area-context/issues/634#issuecomment-3003640605
+        style={this.isAndroid ? { marginTop: initialWindowMetrics?.insets.bottom } : {}}
+        initialSafeAreaInsets={initialWindowMetrics?.insets}
+      >
```

Comment:
This is needed to prevent the status bar from being too close to the header; however, this also is causing a slight grey background on the status bar.

<img src="https://github.com/user-attachments/assets/83930080-33e8-46c6-bd8d-b22a5d0d08a8" width="50%" height="auto" />

Reply:
Fixed in https://github.com/divvi-xyz/divvi-mobile/pull/229/commits/1ac8aad324cc6071b2b96197dc7dd2427e4769dc!